.drone.yml

```
@@ -451,3 +451,8 @@ services:
     image: gitea/test-openldap:latest
     when:
       event: [ push, tag, pull_request ]
+
+  elasticsearch:
+    image: elastic/elasticsearch:5.6.16
```

tests/drone fails because Elasticsearch does not start, it exits during bootstrap checks with this message:

> 28 | [1]: max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]

Reference: https://drone.gitea.io/go-gitea/gitea/7505/1/6

As this is just a test deployment and not a lot of data will be saved, we can disable the (production) bootstrap checks.

https://www.elastic.co/guide/en/elasticsearch/reference/master/bootstrap-checks.html

```suggestion
    image: elastic/elasticsearch:5.6.16
    environment:
      # Disable production checks because they are not relevant and fail
      ES_JAVA_OPTS: "-Des.enforce.bootstrap.checks=true"
```